packages/design/src/FormManager/FormEdit/components/RichTextPatternEdit/index.tsx

```diff
@@ -50,7 +57,6 @@ const RichTextPatternEdit: PatternEditComponent<RichTextProps> = ({
         ></PatternEditForm>
       ) : (
         <div className="padding-left-3 padding-bottom-3 padding-right-3">
-          <p>{message.patterns.richText.displayName}</p>
```

Comment:
I wouldn't delete this text here. This is the handle for testing, and it also serves a purpose. It's a safeguard against an empty editor not having an obvious clickable target to get back into the edit state.